trade-finance-manager-ui/server/routes/index.test.js

```diff
@@ -18,13 +19,14 @@ describe('routes index', () => {
   });
 
   it('should setup all routes', () => {
-    expect(use).toHaveBeenCalledTimes(7);
+    expect(use).toHaveBeenCalledTimes(8);
     expect(use).toHaveBeenCalledWith('/', loginRoutes);
     expect(use).toHaveBeenCalledWith('/case', validateUser, caseRoutes);
     expect(use).toHaveBeenCalledWith('/deals', validateUser, dealsRoutes);
     expect(use).toHaveBeenCalledWith('/facilities', validateUser, facilitiesRoutes);
     expect(use).toHaveBeenCalledWith('/feedback', feedbackRoutes);
     expect(use).toHaveBeenCalledWith('/thank-you-feedback', feedbackThankYouRoutes);
     expect(use).toHaveBeenCalledWith('/user', userRoutes);
+    expect(use).toHaveBeenCalledWith('/', miscRoutes);
```

Comment:
@Zainzzkk can we rename this to `footer` routes? 
it will only have routes from the footer as far as I remember. 
`misc` is too ambiguous